src/corefx/System.Globalization.Native/calendarData.cpp

```diff
@@ -1,6 +1,7 @@
 //
 // Copyright (c) Microsoft. All rights reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+// Licensed under the MIT license. See LICENSE file in the project root for full
+// license information.
```

Comment:
Not really sure what was going on, I reformatted the line and everything worked out okay.